ansible_base/authentication/social_auth.py

```diff
@@ -179,7 +179,9 @@ def validate(self, serializer, data):
         configuration = data['configuration']
         if not configuration.get('CALLBACK_URL', None):
             if not serializer.instance:
-                slug = generate_authenticator_slug(data['type'], data['name'])
+                if not data.get("slug"):
+                    data["slug"] = generate_authenticator_slug()
```

Comment:
We could skip this if now. Just do `data['slug'] = generate_authenticator_slug(data['slug'])`

Reply:
Will do this.